src/OutputReaders/OutputReaders.jl

```diff
@@ -11,15 +11,17 @@ struct InMemory{I} <: AbstractDataBackend
     index_range :: I
 end
 
-InMemory(; chunk_size = Colon()) = ifelse(chunk_size == Colon(), 
-                                          InMemory("", "", chunk_size), 
-                                          InMemory("", "", 1:chunk_size))
+InMemory(; chunk_size = Colon()) = chunk_size isa Colon ? 
+                                          InMemory("", "", chunk_size) :
+                                          InMemory("", "", 1:chunk_size)
 
 struct OnDisk <: AbstractDataBackend 
     path :: String
     name :: String
 end
 
+OnDisk() = OnDisk("", "")
```

Comment:
What does it mean when you have no path but are `OnDisk`?

Reply:
This is a convenience constructor to allow passing `path` and `name` in the `FieldTimeSeries` along with `OnDisk`. 

aka
```julia 
fts = FieldTimeSeries(path, name; backend = OnDisk())`
```

Reply:
Add `path` and `name` as properties of `FieldTimeSeries`? `path = name = nothing` when there's no corresponding file.